other/world.ts

```diff
@@ -57,7 +57,9 @@ export default class World {
   private width: number;
   private height: number;
   private minSpawnDistance = 10;
-  private maxMoveDistance = 2;
+  private minMoveDistance = 2;
+  private maxMoveDistance = 9;
```

Comment:
I wonder why you picked 9 here and if we should consider using a slightly smaller number, for example, 8, to minimize the probability of it jumping over some food. It's ok for me to keep 9, though.

Reply:
In case, when we have minimal radius 5, i think thats imposible, also i have tested, and decided, that this is the best speed to start.

Reply:
@Dimitrymas,

> In case, when we have minimal radius 5, i think thats imposible,

It's still possible to jump over food if it's on the "periphery" of your bot trajectory.

<img width="593" alt="image" src="https://github.com/user-attachments/assets/cafbb1ab-3bc5-4e46-ad37-5d46395345c7" />

We can address this in a follow-up if we see it's being a big deal.